import-dashboards.py

```diff
@@ -420,16 +420,22 @@ def set_home_dashboard(api_key):
     # Get dashboard information by dashboard slug (name) which is "home-dashboard" in our case
     # This API is different from /api/dashboards/home which returns home dashboard
     r = requests.get('%s/api/dashboards/db/home-dashboard' % (HOST,), headers=grafana_headers(api_key))
-    print ' * "home" dashboard: %r %r' % (r.status_code, r.content)
+    print ' * PMM "home" dashboard: %r %r' % (r.status_code, r.content)
     if r.status_code != httplib.OK:
         # TODO sys.exit(-1)
         return
 
-    res = json.loads(r.content)
+    # Get parameters of current home dashboard
+    h = requests.get('%s/api/dashboards/home' % (HOST,), headers=grafana_headers(api_key))
```

Comment:
Could you please give more verbose name for this response?

Reply:
It has been renamed to homeDashboard